webpack.config.js

```diff
@@ -41,7 +41,7 @@ const getWebpackConfig = (config, version) => {
         'process.env.BUILD_ENV': JSON.stringify(buildEnv),
       }),
       new IgnorePlugin({
-        resourceRegExp: /\/iconv-loader$|\.eot$|\.svg$|\.ttf$|\.woff$|\.gif$/,
+        resourceRegExp: /\/iconv-loader$/,
```

Comment:
Turns out the slick carousel is unable to initialize if the assets with these suffix are ignored.